This is my test of try catch and thrown exception !!